### PR TITLE
feat(output): Adds webpack analyzer

### PR DESCRIPTION
**Pull request by jaysoo**

### CODE REVIEW

File: libs/server/src/lib/utils/stats.ts [hunk 1]
```diff
@@ -147,7 +165,7 @@ export function calculateStatsFromChunks(cs: Chunk[]) {
 /* ------------------------------------------------------------------------------------------------------------------ */
 
 interface ChunkData {
-  id: number;
+  id: number | string;
```
Comment (mrmeku) [inline libs/server/src/lib/utils/stats.ts]: Why both?

File: libs/ui/src/lib/build-status/build-status.component.html [hunk 2]
```diff
@@ -147,6 +147,48 @@ <h3>Problems</h3>
...
+        </div>
+        <div class="actions" fxLayout="row" fxLayoutGap="20px">
+          <mat-select
+            matRipple
+            fxFlex
+            [value]="currentChunkId$ | async"
+            (selectionChange)="handleChunkFileSelection($event)"
+          >
+            <mat-option
+              *ngFor="let chunk of (chunks$ | async)"
```
Comment (mrmeku) [inline libs/ui/src/lib/build-status/build-status.component.html]: Is it worth it to use a trackBy here?

File: libs/ui/src/lib/modules-graph/modules-graph.component.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.svg-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.sequence {
+  width: 600px;
```
Comment (mrmeku) [inline libs/ui/src/lib/modules-graph/modules-graph.component.scss]: Can we make this more responsive? Why 600px?

File: libs/ui/src/lib/modules-graph/modules-graph.component.ts [hunk 4]
```diff
@@ -0,0 +1,245 @@
+import { Component, ElementRef, Input, ViewChild } from '@angular/core';
+import { select, selectAll } from 'd3-selection';
+import { arc } from 'd3-shape';
+import { hierarchy, partition } from 'd3-hierarchy';
+import 'd3-transition';
+import { FormatFileSizePipe } from '../format-file-size.pipe';
+
+interface Chunk {
```
Comment (mrmeku) [inline libs/ui/src/lib/modules-graph/modules-graph.component.ts]: Consider running much of the logic within this component outside of the angular zone. D3 is known for taking a huge performance hit when running inside of a zone

File: libs/server/src/lib/utils/stats.ts (hunk 1 above)
Comment (jaysoo) [reply libs/server/src/lib/utils/stats.ts]: webpack comes back with both... but i can normalize to a string.

File: libs/ui/src/lib/modules-graph/modules-graph.component.scss (hunk 3 above)
Comment (jaysoo) [reply libs/ui/src/lib/modules-graph/modules-graph.component.scss]: It becomes somewhat hard to use when too big or too small. Could consider a min-max range maybe?

File: libs/server/src/lib/utils/stats.ts (hunk 1 above)
Comment (mrmeku) [reply libs/server/src/lib/utils/stats.ts]: Do it!

File: apps/angular-console-e2e/src/integration/tasks.spec.ts [hunk 5]
```diff
@@ -244,7 +244,7 @@ describe('Tasks', () => {
     cy.writeFile('../../tmp/proj/src/app/app.component.spec.ts', PASSING_TESTS);
   });
 
-  it('runs serve task', () => {
+  it.only('runs serve task', () => {
```
Comment (mrmeku) [inline apps/angular-console-e2e/src/integration/tasks.spec.ts]: You don't really want this

Comment (mrmeku) [inline apps/angular-console-e2e/src/integration/tasks.spec.ts]: You don't really want this

**Pull request merged**